skyportal/models.py

```diff
@@ -1758,6 +1758,65 @@ def api_class(self):
     def listener_class(self):
         return getattr(facility_apis, self.listener_classname)
 
+    fields = relationship("InstrumentField")
+    tiles = relationship("InstrumentFieldTile")
+
+
+class InstrumentField(ha.Region, Base):
+    """A multi-order healpix representation of a telescope's field shape,
+    as represented by many InstrumentFieldTiles."""
+
+    tile_class = lambda: InstrumentFieldTile  # pylint: disable=E731 # noqa
```

Comment:
What's going on here exactly? Not sure why this is needed

Reply:
This was how Leo had it implemented in his healpix_alchemy implementation (https://github.com/skyportal/healpix-alchemy/pull/15/files#diff-17de8e8b9e279ac99ba4c78cde70804540753339f30642e0faaa614cb6cd843cR72). Is there a better way?

Reply:
I don't know why you'd need this at all

Reply:
How else would I tell it about the tiles in the Region?